src/layman/layer/geoserver/geoserver_test.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from layman import settings
+from layman.http import LaymanError
+from test import process_client
+
+
+@pytest.mark.usefixtures('ensure_layman')
+def test_check_username_wms():
```

Comment:
This test is about workspace, better to rename it.

Also test for reserve username.